site/templates/header-banner.jet

```diff
@@ -0,0 +1,8 @@
+{{block headerBanner()}}
+  {{showBanner:=config("show_header_banner")=="true"}}
+    {{if showBanner}}
```

Comment:
This block doesn't need to be indented so far lol

Reply:
Seems to be similar with the [footer nav](https://github.com/shift72/core-template/blob/develop/site/templates/footer/nav.jet) haha.

Reply:
Everything in the footer nav is indented correctly.  Your `if` block should be aligned with `showBanner`.